pkgs/top-level/all-packages.nix

```diff
@@ -20401,11 +20398,6 @@ with pkgs;
   glew110 = callPackage ../development/libraries/glew/1.10.nix {
     inherit (darwin.apple_sdk.frameworks) AGL OpenGL;
   };
-  glew-egl = callPackage ../development/libraries/glew {
-    inherit (darwin.apple_sdk.frameworks) OpenGL;
-    enableEGL = true;
-  };
-
```

Comment:
I'd start with a `warn` in `aliases.nix`

Reply:
Done